ChronosTankDrive/src/main/java/frc/robot/Robot.java

```diff
@@ -174,10 +171,10 @@ public void teleopPeriodic() {
      * }
      */
     if(accelerationLimiting){
-      accelLimit(rightStick.getY(), leftStick.getY());
+      accelLimit(leftStick.getY(), rightStick.getY());
     }else{
-      rightAdjusted=rightStick.getY();
-      leftAdjusted=leftStick.getY();
+      rightAdjusted=leftStick.getY();
```

Comment:
Why does the leftStick control the right side and the rightStick control the left side?

Reply:
I remember having a reason for that, but I can't remember what it was right now. I'll change this back tomorrow.